flocker/docs/test/test_version_extensions.py

```diff
@@ -18,21 +16,23 @@ def test_version_prompt(self):
         ``|latest-installable|`` in a source file with the current
         installable version in the output file.
         """
-        temp_dir = FilePath(self.mktemp())
-        temp_dir.makedirs()
-        source_file = temp_dir.child('contents.rst')
+        source_directory = self.make_temporary_directory()
+        source_file = source_directory.child('contents.rst')
```

Comment:
Latest sphinx exits with an error if the source and destination directories are the same.